lambda/writePass/index.js

```diff
@@ -72,6 +76,63 @@ exports.handler = async (event, context) => {
       numberOfGuests = 1;
     }
 
+    // Get current time vs booking time information
+    const today = new Date();
+    const currentDay = parseInt(today.toLocaleString('en-US', { day: '2-digit', timeZone: 'America/Vancouver' }));
+    const currentHour = parseInt(
+      today.toLocaleString('en-US', { hour: '2-digit', hour12: false, timeZone: 'America/Vancouver' })
+    );
+    const bookingDay = new Date(date).getDate();
+
+    let facilityObj = {
+      TableName: TABLE_NAME
+    };
+
+    // check if booking date in the past
+    if (currentDay > bookingDay) {
```

Comment:
I think at a month boundary this will fail, e.g. on the 1st you can book for the 31st of last month. Should be able to compare the whole date though.

Reply:
Good catch. Improved in `9408cb3`

Reply:
Fixed in `bb5176b`